shop.py

```diff
@@ -217,13 +218,12 @@ def update_shopify(self):
             shipment.state = 'draft'
             shipment.json_order = order.to_json()
             shipment.customer_phone_numbers = get_customer_phone_numbers(order)
-            shipment.comment = order.note
+            shipment.comment = order.note or ''
+            attributes = []
             for attribute in order.note_attributes:
-                for key, value in attribute.to_dict().items():
-                    if shipment.comment:
-                        shipment.comment += value + '\n'
-                    else:
-                        shipment.comment = value + '\n'
+                attributes += list(attribute.to_dict().values())
+                if shipment.comment:
```

Comment:
Com que saps que shipment.comment no és NULL (has fet or '') not tens per què fer aquest if.